src/GraphQL/Types/Composite/AutoRegisteringObjectGraphType.cs

```diff
@@ -11,7 +11,6 @@ namespace GraphQL.Types
     /// Supports <see cref="DescriptionAttribute"/>, <see cref="ObsoleteAttribute"/>, <see cref="DefaultValueAttribute"/> and <see cref="RequiredAttribute"/>.
     /// Also it can get descriptions for fields from the XML comments.
     /// </summary>
-    /// <typeparam name="TSourceType"></typeparam>
```

Comment:
Then remove it for `AutoRegisteringInputObjectGraphType` too.

Reply:
Done